docs/source/contributing/development.rst

```diff
@@ -88,11 +90,11 @@ For first-time contributors
 
       poetry run pre-commit install
 
-   This will ensure during development that each of your commits is properly
+   This will ensure during development that each of your commits are properly
```

Comment:
each ... is, or alternatively something like all ... are, but I don't agree with this proposed change as is.